dev-utils/storybook/stories/react-components/example.stories.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { withDesign } from "storybook-addon-designs";
+import {
+    Tabs,
+    Tab,
+} from "../../../../feature-utils/poly-look/src/react-components/tabs.jsx";
```

Comment:
Can we maybe use an `@` path for this, defind in package.json, to first avoid such a long path and then having it defined in a single place?

Reply:
I didn't knew you could do that. I will try.

Reply:
This has an important side effect! It would not load the file from the local working directory, but rather as a _package_. That means two things:

1. You need to add _poly-look_ as a dependency to this package here in _dev-util_
2. Whenever you change something in _poly-look_ and want to test it here, you need to rebuild _poly-look_

If we just move the whole thing (back) into poly-look, we don't have this issue.

Reply:
Just a single point "the whole thing" === the whole `stories` directory, right?

Reply:
Aaa, well, in this case, it would be a better developer experience to leave the long paths. Once you start Storybook, you don't need to use any other command, regardless on where you make changes.